src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { OfferData } from 'pages/OfferNew/types'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { getHouseholdSize } from '../utils'
+
+type Props = {
+  offerData: OfferData
+}
+
+const Container = styled.div`
+  width: 100%;
+  padding: 2rem 0;
+  color: ${colorsV3.white};
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    width: auto;
+    padding: 0;
```

Comment:
Is there a reason we set width here? Couldn't see any layout change without it :)

Reply:
Yes, the reason is that if you make the screen smaller this element pushes the CTA box outside of the screen if it has `width: 100%`. Like this:
![image](https://user-images.githubusercontent.com/42962286/104355038-6061fb00-550a-11eb-9d7e-1ce9f3f307cf.png)

Reply:
Definitely! I meant that it looks the same when removing `width: 100%` as well on line 14, in smaller screens that is. But if there's a reason, omit this comment 🙌

Reply:
Ah yes, you're right! Fixing that. Thank you! ✨